src/sentry/static/sentry/app/views/settings/incidentRules/ruleForm/index.tsx

```diff
@@ -219,22 +319,22 @@ class RuleFormContainer extends AsyncComponent<Props, State> {
 
   /**
    * Callback for when triggers change
+   *
+   * Re-validate triggers on every change and reset indicators when no errors
    */
-  handleChangeTriggers = (triggers: Trigger[]) => {
+  handleChangeTriggers = (
+    triggers: Trigger[],
```

Comment:
`Trigger[]` or `Array<Trigger> `? I believe we don't have a rule for that

Reply:
We generally do `Trigger[]`, but I think sometimes `Array<T>` is easier to read if `T` is a more visually complex type.